core/primitives/src/runtime/migration_data.rs

```diff
@@ -0,0 +1,20 @@
+#[cfg(feature = "protocol_feature_fix_storage_usage")]
+use crate::types::AccountId;
+#[cfg(feature = "protocol_feature_fix_storage_usage")]
+use crate::types::Gas;
+use std::fmt;
+use std::fmt::{Debug, Formatter};
+
+#[derive(Default)]
+pub struct MigrationData {
+    #[cfg(feature = "protocol_feature_fix_storage_usage")]
+    pub storage_usage_delta: Vec<(AccountId, u64)>,
```

Comment:
I would prefer to have a struct with named arguments as `u64` is not really descriptive

Reply:
If we use named structure in json this will lead to additional 70kb of data included executable